test/com/google/enterprise/adaptor/ad/AdAdaptorTest.java

```diff
@@ -1184,7 +1184,8 @@ public void testGroupCatalogMakeDefsWellKnownParent() throws Exception {
         assertTrue("results did not have key " + key, results.containsKey(key));
         assertNull("non-null result for key " + key, results.get(key));
       } else {
-        assertEquals(value, results.get(key));
+        // Compare the lists for unordered equality, ignoring duplicates.
```

Comment:
I don't think so. In the tests the data sets are tiny, and in this case the lists have just 2 items, but I wanted to keep it generic for possible reuse elsewhere if necessary later.